aiven/provider.go

```diff
@@ -398,6 +398,18 @@ func validateDurationString(v interface{}, k string) (ws []string, errors []erro
 	return
 }
 
+// validateStringEnum returns a schema.ValidateFunc that ensures that a string one of a few possible values
```

Comment:
Can you please check if we can reuse `validation.StringInSlice`?

Reply:
good catch we can!